cmd/resize.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+    "errors"
+
+    "github.com/getcarina/carina/console"
+    "github.com/spf13/cobra"
+)
+
+func newResizeCommand() *cobra.Command {
+    var options struct {
+        name     string
+        nodes    int
+        wait     bool
+    }
+
+    var cmd = &cobra.Command{
+        Use:               "resize <cluster-name>",
+        Short:             "Resize a cluster",
+        Long:              "Resize a cluster",
+        Hidden:            true,
```

Comment:
Let's remove this line, so that `carina --help` displays this new command.

I was hiding `grow` because it was only supported on make-swarm (similar to autoscale). Since make-swarm support was temporary, any commands specific to make-swarm are hidden.